arm-datalake-analytics/catalog/2015-10-01-preview/swagger/catalog.json

```diff
@@ -976,6 +1002,153 @@
         }
       }
     },
+    "/catalog/usql/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/partitions/{partitionName}": {
+      "get": {
+        "tags": [
+          "Catalog"
+        ],
+        "operationId": "Catalog_GetTablePartition",
+        "description": "Retrieves the specified table partition from the Data Lake Analytics catalog.",
+        "parameters": [
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database containing the partition."
+          },
+          {
+            "name": "schemaName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the schema containing the partition."
+          },
+          {
+            "name": "tableName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the table containing the partition."
+          },
+          {
+            "name": "partitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the table partition."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/USqlTablePartition"
+            }
+          }
+        }
```

Comment:
This will get deserialized [correctly in node.js as the mapper has it defined](https://github.com/Azure/autorest/blob/master/ClientRuntimes/NodeJS/ms-rest-azure/lib/cloudError.js#L60).
It looks like this will be [deserialized correctly in C#] as well(https://github.com/Azure/autorest/blob/master/ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime.Azure/CloudError.cs) (even when the default schema is not specified).
@markcowl @hovsepm - Is my understanding correct?